public/js/main.js

```diff
@@ -34,40 +34,78 @@
   var copyContainer = $('<div/>');
   copyEl.attr('style', 'display: none;');
   copyEl.appendTo('body');
+  var clipboard;
 
   function setupMouseEvents() {
-    $('.library-column').on( "mouseenter", function(ev) {
-      var cont = $(ev.currentTarget);
-      copyEl.show();
-      copyEl.appendTo(cont);
-    })
-    .on( "mouseleave", function(ev) {
-      var cont = $(ev.currentTarget);
-      //copyEl.appendTo('body');
-    });
+    // Currently not showing the copy button for iOS, check clipboard.js support
+    if(!(/iPhone|iPad/i.test(navigator.userAgent))) {
+      $('.library-column').on( "mouseenter", function(ev) {
+        var cont = $(ev.currentTarget);
+        copyEl.show();
+        copyEl.appendTo(cont);
+      })
+      .on( "mouseleave", function(ev) {
+        var cont = $(ev.currentTarget);
+        //copyEl.appendTo('body');
+      });
+      if (clipboard) clipboard.destroy();
+      setupCopyButton();
+    }
   }
 
-  var client = new ZeroClipboard($(".copy-button"));
-  client.on( "ready", function( readyEvent ) {
-    // ZeroClipboard is enabled. Display copy button.
-    setupMouseEvents();
-    client.on( "copy", function (event) {
-      var button = $(event.target);
-      var embed = button.attr('data-copy-embed');
-      var url = $('.library-url', button.parents('.library-column')).text();
-      var oldurl = url;
-      if(embed === 'script') {
-        url = '<script type="text/javascript" src="' + url + '"></script>';
-      } else if (embed === 'link') {
-        url = '<link rel="stylesheet" href="' + url + '">';
+  function setupCopyButton() {
+    clipboard = new Clipboard(".copy-button", {
+      text: function(trigger) {
+        var button = $(trigger);
+        var embed = button.attr('data-copy-embed');
+        var url = $('.library-url', button.parents('.library-column')).text();
+        if(embed === 'script') {
+          url = '<script type="text/javascript" src="' + url + '"></script>';
+        } else if (embed === 'link') {
+          url = '<link rel="stylesheet" href="' + url + '">';
+        }
+        return url;
       }
-          ga('send', 'event', 'library', 'copied', button.parents('.library-column').attr('data-lib-name'), 4);
+    });
 
-      var clipboard = event.clipboardData;
-      clipboard.setData( "text/plain", url );
+    clipboard.on("success", function(e) {
+      var button = $(e.trigger);
+      var btContainer = button.parents('.copy-button-group').tooltip({
+        trigger: 'manual',
+        placement: 'bottom',
+        title: 'Copied!'
+      });
+      btContainer.tooltip('show');
+      setTimeout(function(){
+        btContainer.tooltip('hide');
+        btContainer.tooltip('destroy');
+      }, 1000);
+      ga('send', 'event', 'library', 'copied', button.parents('.library-column').attr('data-lib-name'), 4);
```

Comment:
Hmm, I did not know how to proceed on that matter. If you must log even on the `error` event we could just add the `ga` function call there.